Mathlib/AlgebraicGeometry/Morphisms/RingHomProperties.lean

```diff
@@ -265,7 +265,6 @@ theorem sourceAffineLocally_isLocal (h₁ : RingHom.RespectsIso @P)
 variable (hP : RingHom.PropertyIsLocal @P)
 
 -- Porting note: the terms here are getting huge ~ 1/2 Gb for the goal midway (with `pp.explicit`)
-set_option maxHeartbeats 4000000 in
 theorem sourceAffineLocally_of_source_open_cover_aux (h₁ : RingHom.RespectsIso @P)
     (h₃ : RingHom.OfLocalizationSpanTarget @P) {X Y : Scheme} (f : X ⟶ Y) (U : X.affineOpens)
     (s : Set (X.presheaf.obj (op U.1))) (hs : Ideal.span s = ⊤)
```

Comment:
In #7747 by the original author @erdOne this was golfed (apparently using just one new lemma introduced).

Reply:
Ah, I wish I knew that beforehand :( I'd be happy to have my hacky proof replaced by a proper one, of course.